doc/ir-semantics.text

```diff
@@ -100,49 +111,62 @@ Update (partial) function, returns a S:
 
 ## Reduction relation '-->'
 
-  (P, T, H, E*, C* C) --> M
-    when M := step P(C) (P, T, H, E*, C* C)
+  (P, T, H, E*, π* π) --> M
+    when M := step P(π) (P, T, H, E*, π* π)
 
   step   CC[e]
-         (P, T, H, E*, C*)
-       = step CC[v] (P, T, H, E*, C*)
+         (P, T, H, E*, π*)
+       = step CC[v] (P, T, H, E*, π*)
     when v := eval P H E e
 
+  step   CC[call f (v, ...)]
+         (P, T, H, E*, π*)
+       = (P, T, H, E* E', π* π)
+    when P(f)  = Function (x, ...)
+     and E'   := (x ↦ v), ...
+     and π    := (f, active, start)
```

Comment:
the idea is that by definition active is the most up-to-date version of the function and start is the first instruction